alibuild_helpers/build.py

```diff
@@ -67,6 +67,7 @@ def getDirectoryHash(d):
 
 # Helper class which does not do anything to sync
 class NoRemoteSync:
+  writeStore = ""
```

Comment:
This is just some cleanup, right? Can you do it in a separate PR?

Reply:
I've undone this.